components/Tile/index.module.scss

```diff
@@ -0,0 +1,75 @@
+@import "styles/index";
+
+$shadow: 0 2px 22px 0 rgba(0, 0, 0, 0.15);
+
+.tile {
+  position: relative;
+  overflow: hidden;
+  display: inline-flex;
+  flex-direction: column;
+  background-color: white;
+  width: 220px;
```

Comment:
If this is the same tile/card that we are using in our WP sites, having a fixed width of 220px may be too opinionated for this component. IMO these types of components should be fluid and fill their parent layout component. We can still add a max-width though.